package-lock.json

```diff
@@ -25,6 +25,7 @@
         "@types/react": "^18.2.20",
         "@types/react-dom": "^18.2.7",
         "@typescript-eslint/eslint-plugin": "^6.7.4",
+        "@wixc3/react-board": "^4.0.0",
```

Comment:
why this was changed in this PR?

Reply:
Do you mean version? 
I think Mika has just installed it, not copied from another template.